File: src/problems_all/58_length_of_last_word/length_of_last_word.java
/*
Runtime:    0 ms
Beats:      100.00%
Memory:     41.80 MB
Beats:      33.76%
Submission: https://leetcode.com/problems/length-of-last-word/submissions/1440847575/
*/

class Solution {
    public int lengthOfLastWord(String s) {
        String[] words = s.split(" ");
        return words[words.length - 1].length();
    }

    public static void main(String[] args) {
        String s = "world";
        Solution solution = new Solution();
        int result = solution.lengthOfLastWord(s);
        System.out.println(result);
    }
}

File: src/problems_all/58_length_of_last_word/length_of_last_word.py
"""
Runtime:    0 ms
Beats:      100.00%
Memory:     16.56 MB
Beats:      72.19%
Submission: https://leetcode.com/problems/length-of-last-word/submissions/1440795215/
"""


class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        sentence = s.strip().split(' ')
        if len(sentence) > 0:
            return len(sentence[-1])

        return 0


if __name__ == '__main__':
    s = "Hello World"
    solution = Solution()
    result = solution.lengthOfLastWord(s)
    print(result)
